src/ol/source/imagewmssource.js

```diff
@@ -312,7 +312,7 @@ ol.source.ImageWMS.prototype.getRequestUrl_ = function(extent, size, pixelRatio,
   }
   params['BBOX'] = bbox.join(',');
 
-  return goog.uri.utils.appendParamsFromMap(this.url_, params);
+  return this.url_ + '?' + ol.ext.querystring.stringify(params);
```

Comment:
goog.uri.utils.appendParamsFromMap is pretty hairy... I will see if I can find any suitable replacement.